sys-process/oomd/oomd-0.5.0-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info meson
+
+DESCRIPTION="Userspace Out-Of-Memory (OOM) killer"
+HOMEPAGE="https://github.com/facebookincubator/oomd"
+SRC_URI="https://github.com/facebookincubator/oomd/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="systemd test"
+RESTRICT="!test? ( test )"
+DEPEND="dev-libs/jsoncpp:=
+	dev-util/meson:=
+	systemd? ( sys-apps/systemd:= )
+	test? ( dev-cpp/gtest:= )"
+RDEPEND="${DEPEND}"
+
+CONFIG_CHECK="CGROUPS PSI"
+
+src_configure() {
+	local emesonargs=(
+		$(meson_feature systemd libsystemd)
+	)
+	meson_src_configure
+}
+
+src_install() {
+	meson_src_install
+	einstalldocs
+}
+
+src_test() {
+	meson_src_test
+}
```

Comment:
meson.eclass declares this function as its own, so no need to type this into the ebuild.